src/commands/downlink/SetParameter.ts

```diff
@@ -132,6 +132,22 @@ const examples: TCommandExampleList = [
             data: {state: 1, channel: 2}
         },
         hex: {header: '03 03', body: '1e 01 01'}
+    },
+    {
+        name: 'set pulse channels config',
+        parameters: {
+            id: deviceParameters.PULSE_CHANNELS_SCAN_CONFIG,
+            data: {timePullUp: 18, timeScan: 23}
```

Comment:
how about this change to make names more readable?

```suggestion
            data: {pullUpTime: 18, scanTime: 23}
```